README.md

```diff
@@ -1,2 +1,5 @@
+[![CircleCI](https://circleci.com/gh/giantswarm/valuemodifier.svg?&style=shield&circle-token=2f991b43e9c0147771ebe0575885501a7184dea9)](https://circleci.com/gh/giantswarm/valuemodifier)
+
 # valuemodifier
-Package valuemodifier provides an interface to modify values of arbitrary structures in custom ways. 
+Package valuemodifier provides an interface to modify values of arbitrary
```

Comment:
Although the package is generic I think mentioning the current support is for JSON / YAML would be good. I'd also like to see a brief note on how we will use it.